### ISSUE TITLE
Image resizing

### ISSUE THREAD

**Issue by gbrunner** [darshanbaral/ghazal#3]
@darshanbaral  I like this theme a lot, however, I am having trouble controlling the image size in my posts. It appears that my images are always stretched to fill the page. For example, check out the page here that I'm working on:
https://boomshakalakalaka.z19.web.core.windows.net/geography-posts/multidimensional-data-in-arcgispro/

How do I control the size of the images? It appears the theme isn't responding to anything I do in my markdown.

**Comment by gbrunner** [comment]
I have modified this to some degree by changing the **img** tag:
```
.content img {
  max-width:100%;
  width:auto;
  display: block;
  margin-left: auto;
  margin-right: auto;
}
```

**Comment by darshanbaral** [comment]
<del>I was trying to make the images responsive. Could you tell me more about your desired behavior for the images when the window width becomes narrow?</del>

**Comment by darshanbaral** [comment]
@gbrunner, thanks for pointing it out. I fixed it in 43aeeccd356bf344d13a89414c9ee706c36c9dbb.

**Comment by gbrunner** [comment]
@darshanbaral Awesome. Thank you for addressing this! Again, I like the theme a lot. Check out my live use of your theme at https://www.gbrunner.me/
